contracts/src/spaces/facets/membership/metadata/MembershipMetadata.sol

```diff
@@ -13,6 +14,16 @@ import {ERC721ABase} from "contracts/src/diamond/facets/token/ERC721A/ERC721ABas
 import {Facet} from "contracts/src/diamond/facets/Facet.sol";
 
 contract MembershipMetadata is ERC721ABase, Facet {
```

Comment:
create interface for MembershipMetadataBase

Reply:
Added.